Examples/Playground/Hyperspace.playground/Contents.swift

```diff
@@ -10,17 +10,27 @@ PlaygroundPage.current.needsIndefiniteExecution = true
 // MARK: - Models
 
 struct User: Decodable {
-    let id: Int
+    let identifier: Int
```

Comment:
I'm guessing you changed this to fix a SwiftLint warning, but I'm puzzled as to why the exclusion we have for `identifier_name` in the root `.swiftlint.yml` did not allow this to go through.

Reply:
I did originally, but thought I'd reverted them all. This must have slipped through.